ocaml/idl/datamodel.ml

```diff
@@ -857,6 +859,8 @@ let _ =
     ~doc:"The hosts in this pool are not compatible." ();
   error Api_errors.pool_hosts_not_homogeneous [ "reason" ]
     ~doc:"The hosts in this pool are not homogeneous." ();
+  error Api_errors.pool_management_cannot_be_set_on_system_network [ "network" ]
+    ~doc:"You tried to configure pool management on system network: It is not allowed." ();
```

Comment:
But is this also enforced when calling the `Host.management_configure` API call?
I would have thought that if a network can be set on every host through calls to `Host.management_reconfigure` then it should also be possible to do the same thing through `Pool.management_reconfigure` (and vice versa).